curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63bf45ce0dc8d4270760c6d0.md

```diff
@@ -7,24 +7,26 @@ dashedName: step-22
 
 # --description--
 
-Within your loop, you need to check if the character in `strArray` at index `i` is not a `+`, `-`, or a space. If it is not, `push` it to the `cleanStrArray`.
```

Comment:
Also, in the future, you will want to avoid making changes and PR's off of your main branch.

It is best practice to create a new branch when you want to make changes.
That way you keep your main branch clean and delete the new branch when the changes have been approved and merged into the project.

This will be especially helpful when you start working on larger features and changes on team projects 👍 

You can learn more about branches here
https://www.git-tower.com/learn/git/faq/create-branch